site/en/docs/extensions/reference/storage/index.md

```diff
@@ -132,22 +121,22 @@ The following samples demonstrate the `local`, `sync`, and
 To learn more about the `managed` storage area, see [Manifest for storage areas][manifest-storage].
 
 
-## Storage and throttling limits {: #storage-and-throttling-limits}
+### Storage and throttling limits {: #storage-and-throttling-limits}
 
 Don't think of adding to the Storage API as putting things in a big truck. Think of adding to
 storage as being like putting something in a pipe. The pipe may have material in it already, and it
 may even be filled. Always assume a delay between when you add to storage and when it is actually
-recorded.
+recorded. <!--I'd like to understand the truck metaphor better, and maybe come up with something more accessible. What are we assuming about how a truck on a road behaves differently from fluid in a pipe?-->
```

Comment:
I believe this predates any of us. @AmySteam or @oliverdunk any ideas for a better metaphor.

Reply:
I think the key ideas are:

- Don't assume storage is free. There is some performance cost to storing things and so be mindful about if you need to store something or if you can store just the important things.
- There are quotas on the storage API and if you are frequently storing things/storing things that are too big you may exceed these and then lose the ability to store things entirely for a period of time.
- Storage isn't always instant, and may take some amount of time to complete.

Reply:
Okay, in my head all three of those points fit better with the truck metaphor than the pipe metaphor.
I think it's better if we just replace this analogy with exactly what Oliver said (but punctuated to make it slightly less casual).

Reply:
Sounds good.

Reply:
Updated. Thoughts about the phrasing?